core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3883,7 +3897,24 @@ public synchronized void doSubmitDescription( StaplerRequest req, StaplerRespons
     @RequirePOST
     public synchronized HttpRedirect doQuietDown() {
         try {
-            return doQuietDown(false,0);
+            return doQuietDown(false, 0, null);
+        } catch (IOException | InterruptedException e) {
+            throw new AssertionError(); // impossible
+        }
+    }
+
+    /**
+     * Quiet down Jenkins - preparation for a restart
+     * Presented for compatibility.
+     *
+     * @param block Block until the system really quiets down and no builds are running
+     * @param timeout If non-zero, only block up to the specified number of milliseconds
+     * @deprecated TODO
```

Comment:
```suggestion
     * @deprecated since TODO; use {@code #doQuietDown(boolean,int,String)} instead.
```

(Untested Javadoc, should be verified in IDE.)

Reply:
- [ ] Check proposed javadoc